pkg/metrics/exporters/stackdriver/stackdriver.go

```diff
@@ -0,0 +1,43 @@
+package stackdriver
+
+import (
+	"context"
+	"flag"
+
+	traceapi "cloud.google.com/go/trace/apiv2"
+	"contrib.go.opencensus.io/exporter/stackdriver"
+	"contrib.go.opencensus.io/exporter/stackdriver/monitoredresource"
+	"golang.org/x/oauth2/google"
+)
+
+const Name = "stackdriver"
+
+var ignoreMissingCreds = flag.Bool("stackdriver-only-when-available", false, "Only attempt to start the stackdriver exporter if credentials are available")
+
+const metricPrefix = "custom.googleapis.com/opencensus/gatekeeper/"
+
+func Start(ctx context.Context) error {
+	// Verify that default stackdriver credentials are available
+	if _, err := google.FindDefaultCredentials(ctx, traceapi.DefaultAuthScopes()...); err != nil {
+		if *ignoreMissingCreds {
+			return nil
```

Comment:
Let's add a warning statement telling users the credential is missing but we are ignoring it.

Reply:
added